### PR TITLE
Visual Dev Icon

### PR DESCRIPTION
**Pull request by kutaycinar**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

 - Jira issue number: (PUT IT HERE)
 - Process.st launch checklist: (PUT IT HERE)

### Development

- [ ] [Prettier](https://www.npmjs.com/package/prettier) was run
- [ ] The behaviour changes in the pull request are covered by specs
- [ ] All tests related to the changed code pass in development

### Paperwork

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has a Jira number
- [ ] This pull request has a Process.st launch checklist
- [ ] "Ready for review" label attached to the PR and reviewers pinged on Slack

### Code review 

- [ ] Changes have been reviewed by at least one other engineer
- [ ] Security impacts of this change have been considered

### CODE REVIEW

File: packages/visual-dev/src/components/Icon/Icon.tsx [hunk 1]
```diff
@@ -1,57 +1,42 @@
...
+  icon: IconKey
+}
+
+interface StyleProps {
+  color?: string
+  size?: 'small' | 'medium' | 'large' | string
+  css?: CSSProp
+}
+
+const default_size = {
```
Comment (noahwc) [inline packages/visual-dev/src/components/Icon/Icon.tsx:18]: Just want to double-check that these sizes are defined in the design system. They seem like they might be slightly arbitrary.

File: packages/visual-dev/src/components/stories/Icon.stories.tsx [hunk 2]
```diff
@@ -1,36 +1,38 @@
...
-export const info = () => <Icon icon={"info"}/>
-export const settings = () => <Icon icon={"settings"}/>
+import React from 'react'
+import { Icon } from '../Icon'
+
+export default {
+  title: 'Components / Icon',
+  component: Icon,
+}
+
```
Comment (noahwc) [inline packages/visual-dev/src/components/stories/Icon.stories.tsx:8]: A couple stories with custom CSS (using the `css` prop) would help round out this story set.

**Review by noahwc — COMMENTED** [review]
Looks pretty good! Just a couple of nitpicks.

File: packages/visual-dev/src/components/Icon/Icon.tsx (hunk 1 above)
Comment (kutaycinar) [reply packages/visual-dev/src/components/Icon/Icon.tsx:18]: Messaged Shirley about this, on hold for now.

File: packages/visual-dev/src/components/stories/Icon.stories.tsx (hunk 2 above)
Comment (kutaycinar) [reply packages/visual-dev/src/components/stories/Icon.stories.tsx:8]: Added one.

image

File: packages/visual-dev/src/components/Icon/Icon.tsx (hunk 1 above)
Comment (kutaycinar) [reply packages/visual-dev/src/components/Icon/Icon.tsx:18]: Updated values as per the new design constraints:

small: 12px (bounding box: 20px)
medium: 20px (bounding box: 20px)
large: 36px (bounding box: 36px)

**Pull request merged**